docker/docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
       - pycharm_config:/root/.PyCharmCE2020.2.1/
       - ros_logs:/root/.ros/log
       - scratch:/root/scratch/
-      - workspace:/root/catkin_ws/
```

Comment:
Probably better not to commit these changes, as it's a workaround to creating a new volume. You can select them not to be included in the commit with `git gui`, by staging only specific lines.